cpg_workflows/large_cohort/scripts/browser_vcf_release.py

```diff
@@ -2214,6 +2220,26 @@ def add_capture_region_flags(ht: hl.Table) -> hl.Table:
     return ht
 
 
+def repartition_frequencies_table(
+    ht_path: str,
+    data_type: str,
+    new_path: str,
+):
+    """
+    Repartition the genomes and joint frequency table.
+
+    If data_type is exome, no need to repartition, return existing path to
+    frequencies table.
+    If data_type is genome or joint, check if can_reuse, otherwise
+    repartition and checkpoint to temp. Return path to this table
```

Comment:
```suggestion
    Repartition the genomes and joint frequency tables, this method is not required for Exomes.
    
    First use can_reuse to check if the new data path exists. If not repartition and checkpoint.
```

I forgot to change this